llama-index-integrations/callbacks/llama-index-callbacks-arize-phoenix/pyproject.toml

```diff
@@ -20,16 +20,20 @@ ignore_missing_imports = true
 python_version = "3.8"
 
 [tool.poetry]
-authors = ["Your Name <you@example.com>"]
+authors = [
+    {email = "oss@arize.com", name = "OpenInference Authors"},
+    {email = "phoenix-devs@arize.com", name = "Arize AI"},
+]
 description = "llama-index callbacks arize-phoenix integration"
 license = "MIT"
 name = "llama-index-callbacks-arize-phoenix"
 readme = "README.md"
-version = "0.1.1"
+version = "0.1.2"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<3.12"
-arize-phoenix = "^2.5.0"
+arize-phoenix = ">=3.0.3"
+openinference-instrumentation-llama-index = ">=1.0.0"
```

Comment:
This might just end up locking up the versions.  I guess we can try that for now?

Reply:
updated poetry.lock